lib/player/player.js

```diff
@@ -736,6 +741,34 @@ shaka.player.Player.prototype.setPlaybackRate = function(rate) {
 
 
 /**
+ * @param {shaka.player.DrmSchemeInfo.Restrictions} restrictions
+ *     Restrictions object instance with custom restricions.
+ *     For example: A Restrictions instance with maxBandwidth = 700000
+ *     and minBandwidth = 200000 will restrict
+ *     playback to video tracks with bandwidth between 700000 and 200000.
+ * @export
+ */
+shaka.player.Player.prototype.setRestrictions = function(restrictions) {
+  shaka.asserts
+    .assert(restrictions instanceof shaka.player.DrmSchemeInfo.Restrictions);
```

Comment:
The reason I added these checks was to be consistent with methods such as setPlaybackRate() above. Because these restrictions can be set at the application level, I don't see what's preventing someone from passing in an object literal which will then cause problems if they try to getRestrictions().